scribble-doc/scribblings/scribble/acmart.scrbl

```diff
@@ -123,38 +127,38 @@ screen version of the image links to the badge authority.
 
 }
 
-@defproc[(affiliation [content pre-content?] ...) content?]{
-
-Declares information about the affiliation of an author.}
-
-@deftogether[(
-@defproc[(position [content pre-content?] ...) content?]
-@defproc[(institution [content pre-content?] ...) content?]
-@defproc[(department [content pre-content?] ...) content?]
-@defproc[(streetaddress [content pre-content?] ...) content?]
-@defproc[(city [content pre-content?] ...) content?]
-@defproc[(state [content pre-content?] ...) content?]
-@defproc[(postcode [content pre-content?] ...) content?]
-@defproc[(country [content pre-content?] ...) content?]
-)]{
-
-Declares information that is collected for each author.  These commands should
- only be used within an @racket[affiliation] command.}
+@defproc[(affiliation
+          [#:position position (or/c string? #f) #f]
+          [#:institution institution (or/c string? #f) #f]
+          [#:department department (or/c string? #f) #f]
+          [#:street-address street-address (or/c string? #f) #f]
+          [#:city city (or/c string? #f) #f]
+          [#:state state (or/c string? #f) #f]
+          [#:postcode postcode (or/c string? #f) #f]
+          [#:country country (or/c string? #f) #f])
+         affiliation?]{
+ Creates an affiliation object for use with @racket[author].
+}
 
 @codeblock|{
   #lang scribble/acmart
   @title{Some Title}
-  @authorinfo["David Van Horn"
-              @affiliation[
-               #:department "Department of Computer Science and UMIACS"
-               #:institution "University of Maryland"
-               #:city "College Park"
-               #:state "Maryland"]
-              "dvanhorn@cs.umd.edu"]}
+  @author["David Van Horn"
+          #:affiliation @affiliation[
```

Comment:
This would be a good place to demonstrate multiple departments (I should have done it the first time around) since I have two.